RyanNoelk/OpenEats
# Grocery List

To test:
- Update python requirements
- Run migrations
- npm install
- Log in
- Navigate to My Account -> Grocery List

My goal is that the UI should be simple and easy enough for anyone to use and understand. If you're not understanding anything let me know and I will try and explain/fix it. I'm still missing translations for this. But we can add them once this gets merged in.
